### ISSUE TITLE
[WoTLKC] Bugs after last game patch

### ISSUE THREAD

**Issue by Iowerth** [Dathwada/handynotes-travelguide#23]
Hi. Addon version is WotLKC - v1.2.
After yesterday's game patch there are many errors in BugSack.

A little ones:
```lua
1x ...aceHandyNotes_TravelGuide/core/constants.lua:50: attempt to call global 'GetObjectIconTextureCoords' (a nil value)
[string "@HandyNotes_TravelGuide/core/constants.lua"]:50: in main chunk

Locals:
FOLDER_NAME = "HandyNotes_TravelGuide"
private = <table> {
 constants = <table> {
 }
}
constants = <table> {
 defaults = <table> {
 }
}
(*temporary) = nil
(*temporary) = 4772
(*temporary) = "attempt to call global 'GetObjectIconTextureCoords' (a nil value)"


1x HandyNotes_TravelGuide/core/constants.lua:1 (null)

1x HandyNotes_TravelGuide/core/handler.lua:1 (null)

1x Interface_Wrath\FrameXML\Bindings.xml:1 HandyNotes_TravelGuide/core/constants.lua:1 (null)

1x Interface_Wrath\FrameXML\Bindings.xml:1 HandyNotes_TravelGuide/core/handler.lua:1 (null)	
```

And big one:
```lua
1x ...rfaceHandyNotes_TravelGuide/core/handler.lua:19: attempt to index field 'icon' (a nil value)
[string "@HandyNotes_TravelGuide/core/handler.lua"]:19: in main chunk

Locals:
FOLDER_NAME = "HandyNotes_TravelGuide"
private = <table> {
 locale = <table> {
 }
 constants = <table> {
 }
}
addon = <table> {
 SetDefaultModuleLibraries = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:363
 Enable = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:290
 NewModule = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:231
 constants = <table> {
 }
 EnableModule = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:328
 modules = <table> {
 }
 GetModule = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:206
 IterateEmbeds = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:438
 IsModule = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:464
 SetDefaultModuleState = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:383
 defaultModuleLibraries = <table> {
 }
 UnregisterAllEvents = <function> defined @Ace3/CallbackHandler-1.0/CallbackHandler-1.0.lua:169
 SendMessage = <function> defined @Ace3/CallbackHandler-1.0/CallbackHandler-1.0.lua:54
 SetEnabledState = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:420
 UnregisterMessage = <function> defined @Ace3/CallbackHandler-1.0/CallbackHandler-1.0.lua:148
 enabledState = true
 RegisterMessage = <function> defined @Ace3/CallbackHandler-1.0/CallbackHandler-1.0.lua:85
 RegisterEvent = <function> defined @Ace3/CallbackHandler-1.0/CallbackHandler-1.0.lua:85
 UnregisterAllMessages = <function> defined @Ace3/CallbackHandler-1.0/CallbackHandler-1.0.lua:169
 GetName = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:275
 defaultModuleState = true
 Disable = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:310
 IterateModules = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:433
 name = "HandyNotes_TravelGuide"
 SetDefaultModulePrototype = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:405
 IsEnabled = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:447
 orderedModules = <table> {
 }
 UnregisterEvent = <function> defined @Ace3/CallbackHandler-1.0/CallbackHandler-1.0.lua:148
 DisableModule = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:346
}
HandyNotes = <table> {
 SetDefaultModuleLibraries = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:363
 GetContinentZoneList = <function> defined @HandyNotes/HandyNotes.lua:232
 GetMapFiletoMapID = <function> defined @HandyNotes/HandyNotes.lua:264
 Enable = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:290
 RegisterChatCommand = <function> defined @Ace3/AceConsole-3.0/AceConsole-3.0.lua:81
 EnableModule = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:328
 modules = <table> {
 }
 GetModule = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:206
 IterateEmbeds = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:438
 plugins = <table> {
 }
 defaultModuleLibraries = <table> {
 }
 UnregisterChatCommand = <function> defined @Ace3/AceConsole-3.0/AceConsole-3.0.lua:107
 Printf = <function> defined @Ace3/AceConsole-3.0/AceConsole-3.0.lua:65
 OnProfileChanged = <function> defined @HandyNotes/HandyNotes.lua:659
 SetDefaultModulePrototype = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:405
 UpdateMinimapPlugin = <function> defined @HandyNotes/HandyNotes.lua:429
 name = "HandyNotes"
 IsEnabled = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:447
 orderedModules = <table> {
 }
 DisableModule = <function> defined @Ace3/AceAddon-3.0/AceAddon-3.0.lua:346
 UpdateWorldMapPlugin = <function> defi
```

**Comment by Dathwada** [comment]
Will be updated in the next 24h.

**Comment by Dathwada** [comment]
Just pushed an new release here, on Wago and Curseforge. Should be available in a few minutes.

Note: Issue was already fixed with 43f4aa3a7e86ed5eba84947a914cc673da54a010 but not released for the WotLKC version because the addon for this client was still working.
